README.md

```diff
@@ -424,7 +424,7 @@ If you want to help, send an email to support at rocket.chat to be invited to th
 
 ## How to Contribute
 
-Already a JavaScript developer? Familiar with Meteor? [Pick an issue](https://github.com/RocketChat/Rocket.Chat/labels/contrib%3A%20easy), push a PR and instantly become a member of Rocket.Chat's international contributors community.
+Already a JavaScript developer? Familiar with Meteor? [Pick an issue](https://github.com/RocketChat/Rocket.Chat/labels/contrib%3A%20easy), push a PR and instantly become a member of Rocket.Chat's international contributors community. For more information, check out our [Contributing Guide](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md).
```

Comment:
might be a good idea to make this link relative.  Something like (.github/CONTRIBUTING.md)

Reply:
Oh yea, forgot about that :sweat_smile: